portal-web/test/functional/com/liferay/portalweb/paths/portlet/webcontent/WCEditWebContent.path

```diff
@@ -256,7 +256,7 @@
 <!--REPEATED FIELD ON WEB CONTENT-->
 <tr>
 	<td>REPEATED_FIELD</td>
-	<td>xpath=(//div[contains(@class,'ddm-field')][contains(@data-field-name,'${key_fieldName}')][not(contains(@class,'container'))])[${key_index}]</td>
+	<td>xpath=(//div[contains(@class,'ddm-field')]//label[normalize-space(text())='${key_fieldName}'])[${key_index}]</td>
```

Comment:
Hi @leticia-maciel , when you changed this locator, you broke another 4 tests, so I've used your locator on another PATH. Thanks!